app/controllers/spree/admin/shipping_methods_controller.rb

```diff
@@ -79,13 +79,14 @@ def location_after_save
 
       def load_data
         @available_zones = Zone.order(:name)
+        @tax_categories = Spree::TaxCategory.order(:name)
         @calculators = ShippingMethod.calculators.sort_by(&:name)
```

Comment:
can we replace this line with the following while we are at it?
```suggestion
        @calculators = ShippingMethod.calculators.order(:name)
```

I think the risk of breaking things with this change is very very low, don't you agree?

Reply:
I just tried this and `ShippingMethod.calculators` returns an array, and this doesn't work. We might be able to refactor that method, but it looks a bit messy right now...

Reply:
I see. I thought there might a good reason why `order` wasn't used in the first place. Thanks for investigating.